src/modules/resume/prompt.ts

```diff
@@ -0,0 +1,35 @@
+import * as Handlebars from "handlebars";
+import { cloneDeep } from "lodash";
+
+const _MODEL_NAME_GPT3 = "gpt-3.5-turbo";
+const _MODEL_NAME_GPT4 = "gpt-4o";
+const _MODEL_NAME = _MODEL_NAME_GPT4;
+
+export const getResumePrompt = (text: string) => {
+  const resumePrompt = cloneDeep(_RESUME_PROMPT);
+  resumePrompt.messages[1].content = Handlebars.compile(
+      resumePrompt.messages[1].content
+  )({resume: text, modelName: _MODEL_NAME });
+  return resumePrompt;
+};
+
+const _RESUME_PROMPT = {
```

Comment:
variable should be declared before its usage

Reply:
Moved the _RESUME_PROMPT block above its usage function